addons/mail/static/tests/chatter_tests.js

```diff
@@ -2264,6 +2264,69 @@ QUnit.test('followers widget: do not display follower duplications', async funct
     form.destroy();
 });
 
+QUnit.test('followers widget: display inactive followers with a different style', async function (assert) {
+    assert.expect(2);
+
+    this.data.partner.records[0].message_follower_ids = [1,2,3];
+
+    var resID = 2;
+    var followers = [{
+        id: 1,
+        name: "Admin",
+        email: "admin@example.com",
+        res_id: resID,
+        res_model: 'partner',
+        active: true,
+    },{
+        id: 2,
+        name: "Active_partner",
+        email: "active_partner@example.com",
+        res_id: resID,
+        res_model: 'partner',
+        active: true,
+    },{
+        id: 3,
+        name: "Inactive_partner",
+        email: "inactive_partner@example.com",
+        res_id: resID,
+        res_model: 'partner',
+        active: false,
+    }];
+
+    var def;
+    var form = await createView({
+        View: FormView,
+        model: 'partner',
+        data: this.data,
+        services: this.services,
+        arch: '<form>' +
+                '<sheet></sheet>' +
+                '<div class="oe_chatter">' +
+                    '<field name="message_follower_ids" widget="mail_followers"/>' +
+                '</div>' +
+            '</form>',
+        mockRPC: function (route, args) {
+            if (route === '/mail/read_followers') {
+                return Promise.resolve(def).then(function () {
+                    return {
+                        followers: _.filter(followers, function (follower) {
+                            return _.contains(args.follower_ids, follower.id);
+                        }),
+                        subtypes: [],
+                    };
+                });
+            }
+            return this._super.apply(this, arguments);
+        },
+        res_id: resID,
+        session: {partner_id: 1},
+    });
+
+    assert.strictEqual(form.$('.o_inactive_partner').length, 1, 'there is 1 inactive follower');
+    assert.strictEqual(form.$('.o_inactive_partner_img').length, 1, 'there is 1 inactive follower\'s avatar');
```

Comment:
Would be much better to assert on each follower list item. Could use `[data-oe-id="<id>"]` in the jQuery selector.